config/parameters.yml

```diff
@@ -1,10 +1,10 @@
 application_name: Flyimg.io
 
 #debug
-debug: false
+debug: true
```

Comment:
Yes, I would have it on by default, I assume it's useful especially when first time you setup the application and get an error, so at least you understand why you get it.